src/main/java/it/gov/pagopa/pu/classification/service/AssessmentsServiceImpl.java

```diff
@@ -0,0 +1,71 @@
+package it.gov.pagopa.pu.classification.service;
+
+import it.gov.pagopa.pu.classification.connector.debtposition.DebtPositionTypeOrgService;
+import it.gov.pagopa.pu.classification.connector.debtposition.InstallmentNoPIIService;
+import it.gov.pagopa.pu.classification.connector.processexecutions.IngestionFlowFileService;
+import it.gov.pagopa.pu.classification.enums.AssessmentStatus;
+import it.gov.pagopa.pu.classification.model.Assessments;
+import it.gov.pagopa.pu.classification.repository.AssessmentsRepository;
+import it.gov.pagopa.pu.debtposition.dto.generated.DebtPositionTypeOrg;
+import it.gov.pagopa.pu.debtposition.dto.generated.InstallmentNoPIIResponse;
+import it.gov.pagopa.pu.p4paprocessexecutions.dto.generated.IngestionFlowFile;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+@Lazy
+@Slf4j
+@Service
+public class AssessmentsServiceImpl implements AssessmentsService {
+
+  private final InstallmentNoPIIService installmentNoPIIService;
+  private final IngestionFlowFileService ingestionFlowFileService;
+  private final DebtPositionTypeOrgService debtPositionTypeOrgService;
+  private final AssessmentsRepository assessmentsRepository;
+
+  public AssessmentsServiceImpl(InstallmentNoPIIService installmentNoPIIService, IngestionFlowFileService ingestionFlowFileService, DebtPositionTypeOrgService debtPositionTypeOrgService, AssessmentsRepository assessmentsRepository) {
+    this.installmentNoPIIService = installmentNoPIIService;
+    this.ingestionFlowFileService = ingestionFlowFileService;
+    this.debtPositionTypeOrgService = debtPositionTypeOrgService;
+    this.assessmentsRepository = assessmentsRepository;
+  }
+
+  @Override
+  public List<InstallmentNoPIIResponse> getInstallmentsByReceiptId(Long receiptId, String accessToken) {
+    return installmentNoPIIService.getByReceiptId(receiptId, accessToken);
+  }
+
+  @Override
+  public List<Assessments> createAssesment(Long receiptId, String accessToken) {
+    List<InstallmentNoPIIResponse> installmentsList = this.getInstallmentsByReceiptId(receiptId, accessToken);
```

Comment:
Probably we don't neet a separate method to just execute this single line of code (if you don't already plan to re-use it)